megamek/src/megamek/common/templates/AeroTROView.java

```diff
@@ -172,14 +178,50 @@ protected int addWeaponBays(String[][] arcSets) {
         return nameWidth;
     }
 
+    
+    /**
+     * Computes any modification to the aerospace AV for linked Artemis, Apollo, or PPC capacitor
+     *
+     * @param weapon   The type of weapon
+     * @param linkedBy The type of equipment the weapon is linked by
+     * @param bay      Whether the weapon is part of a weapon bay
+     * @return         The AV modification, if any
+     */
+    private int aeroAVMod(WeaponType weapon, EquipmentType linkedBy, boolean bay) {
+        if (linkedBy.hasFlag(MiscType.F_ARTEMIS)
+                || linkedBy.hasFlag(MiscType.F_ARTEMIS_V)) {
+            // The 9 and 10 rows of the cluster hits table is only different in the 3 column
+            if (weapon.getAtClass() == WeaponType.CLASS_MML) {
+                if (weapon.getRackSize() >= 7) {
+                    return 2;
+                } else if ((weapon.getRackSize() >= 5) || linkedBy.hasFlag(MiscType.F_ARTEMIS_V)) {
+                    return 1;
+                }
+            } else if (weapon instanceof LRMWeapon) {
+                return weapon.getRackSize() / 5;
+            } else if (weapon instanceof SRMWeapon) {
+                return 2;
+            }
+        } else if (linkedBy.hasFlag(MiscType.F_ARTEMIS_PROTO) && weapon.getRackSize() == 2) {
```

Comment:
```suggestion
        } else if (linkedBy.hasFlag(MiscType.F_ARTEMIS_PROTO) && (weapon.getRackSize() == 2)) {
```
[nitpick] you missed the parenthesis here.